scripts/start

```diff
@@ -51,16 +52,19 @@ echo
 echo "Setting environment variables..."
 echo
 
+# Source persisted environment variables.
+. $ENV_FILE
+
 # Whitelist device IP, hostname and hidden service for CORS
 DEVICE_IP="$(ip -o route get to 8.8.8.8 | sed -n 's/.*src \([0-9.]\+\).*/\1/p')"
 DEVICE_HOSTNAME="$(hostname)"
-DEVICE_HOSTS="http://${DEVICE_IP},http://${DEVICE_HOSTNAME}.local,https://${DEVICE_HOSTNAME}.local,http://${DEVICE_HOSTNAME},https://${DEVICE_HOSTNAME}"
+DEVICE_HOSTS="http://${DEVICE_IP},http://${DEVICE_HOSTNAME}${DEVICE_DNS_SUFFIX},https://${DEVICE_HOSTNAME}${DEVICE_DNS_SUFFIX},http://${DEVICE_HOSTNAME},https://${DEVICE_HOSTNAME}"
```

Comment:
```suggestion
DEVICE_HOSTS="http://${DEVICE_IP},http://${DEVICE_HOSTNAME}${DEVICE_DNS_SUFFIX},https://${DEVICE_HOSTNAME}${DEVICE_DNS_SUFFIX},http://${DEVICE_HOSTNAME},https://${DEVICE_HOSTNAME},http://${DEVICE_HOSTNAME}.local,https://${DEVICE_HOSTNAME}.local"
```

Doing it like that should work.

Reply:
Is something like this also needed in lnd's `tlsextradomain` config?

Reply:
I'm not sure, but I think yes

Reply:
Principally, it is not required in `tlsextradomain`, since there the value of `DEVICE_HOSTNAME` should be used, which now is the hostname with the custom DNS suffix, so these are in sync. HOWEVER, since this will affect the LND connect URLs, these will change for users on the first start of the new version, which will break their existing lightning wallet connections, which is bad.

The only solution to this that I can think of is trying to resolve hostname.local in scripts/configure. If this works, use hostname.local everywhere and ignore the custom DNS suffix. This provides a smooth update path for existing users.

Simply adding another tlsextradomain entry to LND's config will make LND accept these connections, but it will change the TLS certificate during the update and thus existing connections will be broken anyways and must be re-established. Thus adding the extra domain does not really help.

Reply:
Unfortunately I could not solve this problem sufficiently. When I ping my umbrel node at `umbrel.local` from the box itself, it works fine (because it has `avahi` installed), but my target machine does not have a mDNS resolver, so it cannot connect.

In the end I got avahi installed on my target machine and can now connect on `umbrel.local`, so I guess this PR is most likely not needed anymore, since the recommendation is to either get a mDNS resolver installed or connect to umbrel through the local IP address instead to simplify support requests.

@AaronDewes what do you think?

Reply:
Oh sorry, forgot to respond, I don't think this PR is needed anymore, but @lukechilds and @mayankchhabra schould decide that.